openassessment/xblock/static/js/src/oa_server.js

```diff
@@ -26,57 +22,45 @@ if (typeof OpenAssessment.Server === "undefined" || !OpenAssessment.Server) {
     OpenAssessment.Server.prototype = {
 
         /**
-        Construct the URL for the handler, specific to one instance of the XBlock on the page.
-
-        Args:
-            handler (string): The name of the XBlock handler.
-
-        Returns:
-            URL (string)
-        **/
+         * Returns the URL for the handler, specific to one instance of the XBlock on the page.
+         *
+         * @param {string} handler The name of the XBlock handler.
+         * @returns {*} The URL for the handler.
+         */
         url: function(handler) {
             return this.runtime.handlerUrl(this.element, handler);
         },
 
         /**
-        Render the XBlock.
-
-        Args:
-            component (string): The component to render.
-
-        Returns:
-            A JQuery promise, which resolves with the HTML of the rendered XBlock
-            and fails with an error message.
-
-        Example:
-            server.render('submission').done(
-                function(html) { console.log(html); }
-            ).fail(
-                function(err) { console.log(err); }
-            )
-        **/
+         * Render the XBlock.
+         *
+         * @param {string} component The component to render.
+         * @returns {*} A JQuery promise, which resolves with the HTML of the rendered XBlock
+         *     and fails with an error message.
+         */
         render: function(component) {
             var view = this;
             var url = this.url('render_' + component);
             return $.Deferred(function(defer) {
                 $.ajax({
-                    url: url,
-                    type: "POST",
-                    dataType: "html"
-                }).done(function(data) {
-                    defer.resolveWith(view, [data]);
-                }).fail(function() {
-                    defer.rejectWith(view, [gettext('This section could not be loaded.')]);
-                });
+                        url: url,
```

Comment:
Nit: I'm not sure why you changed the spacing in these blocks. The way it was before looks right to me.